quickstarts/observability-as-code/ansible/config.yml

```diff
@@ -0,0 +1,50 @@
+id: c377d781-da77-4dd6-b5a8-6a25a39e2582
```

Comment:
Thanks for the contribution! I'm not sure how this change to move a config to another directory will impact our downstream service where quickstarts are stored, so let me check with one of our engineering teams on the impact of this change.